Assets/BidMachine/Platforms/Dummy/DummyBidMachine.cs

```diff
@@ -133,6 +133,14 @@ public IAdRequestBuilder SetListener(IAdRequestListener listener)
             return null;
         }
 
+        public IAdRequestBuilder SetRequestListener(IAdAuctionRequestListener listener)
```

Comment:
I'm sure that we can use method overload of the `SetListener` method instead of creating another name for the method. The API will be cleaner.

Reply:
✅